src/rpm_ostree/cli_status.rs

```diff
@@ -14,6 +15,25 @@ use std::fs;
 /// of the current state of deployments.
 const OSTREE_DEPLS_PATH: &str = "/ostree/deploy";
 
+lazy_static::lazy_static! {
+    static ref DEPLOYMENTS: IntGauge = register_int_gauge!(opts!(
+        "zincati_deployments_count",
```

Comment:
Actually thinking about this a bit more, would this metric be useful? I think in most cases it would be `1`, since rpm-ostree saves 1 other deployment by default; but this metric doesn't really tell us any useful information, like for example whether there has likely been a rollback (this information would be more accurately predicted by the `zincati_cincinnati_update_targets_ignored_total` metric from https://github.com/coreos/zincati/pull/456/files).
I think that, at best, a high `zincati_deployments_count` could indicate to us that a user may be doing manual deployments with `rpm-ostree deploy` behind Zincati's back (or perhaps just configured rpm-ostree to save many deployments)? What do you think @lucab?

Reply:
> but this metric doesn't really tell us any useful information

I agree with that, "how many target updates are we ignoring" is the real interesting signal.

Reply:
I've removed that metric then